python/test/unit/language/test_core.py

```diff
@@ -6422,6 +6422,70 @@ def test_convert2d(M, N, src_layout, interm_layout, dst_layout, dtype, device, t
     torch.testing.assert_close(z, x, rtol=0, atol=0)
 
 
+src_layouts = [BlockedLayout([1, 1], [1, 64], [1, 1], [0, 1], [1, 1], [1, 1], [0, 1])]
+
+dst_layouts = [
+    LinearLayout([[0, 32]], [[0, 1], [0, 2], [0, 4], [0, 8], [0, 16], [1, 0]], [], []),
+    LinearLayout([[1, 0], [0, 32]], [[0, 1], [0, 2], [0, 4], [0, 8], [0, 16], [2, 0]], [], []),
+    LinearLayout([[1, 0], [2, 0], [0, 32]], [[0, 1], [0, 2], [0, 4], [0, 8], [0, 16], [4, 0]], [], [])
+]
+
+
+@pytest.mark.parametrize("M, dst_layout", [[2, dst_layouts[0]], [4, dst_layouts[1]], [8, dst_layouts[2]]])
+@pytest.mark.parametrize("src_layout", [src_layouts[0]])
```

Comment:
This can be simplified into `@pytest.mark.parametrize("src_layout", src_layouts)`?